release/create-release-candidate-branch.sh

```diff
@@ -4,37 +4,40 @@
 #
 set -euo pipefail
 set -x
-#-----------------------------------------------------------
+
 # tags should be semver-compatible e.g. 23.1.1 not 23.01.1
 # this is needed for cargo commands to work properly
-#-----------------------------------------------------------
-TAG_REGEX="^[0-9][0-9]\.([1-9]|[1][0-2])\.[0-9]+$"
+# optional release-candidate suffixes are in the form:
+#	- rc-1, e.g. 23.1.1-rc1, 23.12.1-rc12 etc.
+TAG_REGEX="^[0-9][0-9]\.([1-9]|[1][0-2])\.[0-9]+(-rc[0-9]+)?$"
 REPOSITORY="origin"
+PR_MSG="> [!CAUTION]
+> ## DO NOT MERGE MANUALLY!
+> This branch will be merged (and the commit tagged) by stackable-utils once any necessary commits have been cherry-picked to here from the main branch."
 
-tag_products() {
+rc_branch_products() {
 	# assume that the branch exists and has either been pushed or has been created locally
 	cd "$TEMP_RELEASE_FOLDER/$DOCKER_IMAGES_REPO"
-	#-----------------------------------------------------------
-	# the release branch should already exist
-	#-----------------------------------------------------------
-	git switch "$RELEASE_BRANCH"
+
+	# the PR branch should already exist
+	git switch "$PR_BRANCH"
 	update_product_images_changelogs
 
 	git commit -sam "release $RELEASE_TAG"
```

Comment:
Oh, something I have had on my list is to use conventional commit messages for genreated commits.

Reply:
Oh, also PR titles (which end up as the commit message) once squash-merged.

Reply:
See 81a0aa9